### DISCUSSION TITLE
What should I do if I forget my  password of admin

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#764] [Q&A] [answered]
Where can I reset my password？

**Comment by donggyu04** [comment]
@zhougy1989 

Hi, let's following the steps.

1. Log in as admin user.
2. Open `User Management` menu.
3. Click the `edit` button of the user you want to edit.
4. Change the password.

<img src="https://user-images.githubusercontent.com/14273601/108942302-15efa480-769a-11eb-942a-f244bad83551.png" width="150" height="300" />

**Reply by zhougy1989** [reply]
@donggyu04 
but i can not login by admin

**Comment by donggyu04** [comment, marked as answer]
@zhougy1989 

If you forget `admin` password.
Set `controller.admin_password_reset=true` in system configuration and restart the controller.

> admin password will be changed to `admin`

**Reply by junoyoon** [reply]
@zhougy1989 see above.
You can set ```controller.admin_password_reset=true``` by editing ```~/.ngrinder/system.conf```

**Reply by zhougy1989** [reply]
ok,it work ,thks
